lib/tests/test_ancestry.c

```diff
@@ -1378,25 +1381,33 @@ test_gc_tract_lengths(void)
     size_t j;
 
     for (j = 0; j < sizeof(tract_lengths) / sizeof(double); j++) {
-        run_gc_simulation(10, 1.0, tract_lengths[j], 0.1, true);
-        run_gc_simulation(10, 1.0, tract_lengths[j], 0.1, false);
+        run_gc_simulation(10, 1.0, tract_lengths[j], 0.1, true, false);
+        run_gc_simulation(10, 1.0, tract_lengths[j], 0.1, false, false);
+        run_gc_simulation(10, 1.0, tract_lengths[j], 0.1, true, true);
+        run_gc_simulation(10, 1.0, tract_lengths[j], 0.1, false, true);
     }
 }
 
 static void
 test_gc_zero_recombination(void)
 {
-    run_gc_simulation(10, 1.0, 5, 0.0, true);
-    run_gc_simulation(10, 1.0, 1, 0.0, false);
+    run_gc_simulation(10, 1.0, 5, 0.0, true, false);
+    run_gc_simulation(10, 1.0, 1, 0.0, false, false);
+    run_gc_simulation(10, 1.0, 5, 0.0, true, true);
+    run_gc_simulation(10, 1.0, 1, 0.0, false, true);
 }
 
 static void
 test_gc_rates(void)
 {
-    run_gc_simulation(10, 0.1, 1.5, 1.0, false);
-    run_gc_simulation(5, 5.0, 2.5, 0.01, false);
-    run_gc_simulation(10, 1, 1, 1.0, false);
-    run_gc_simulation(30, 1.0, 6.5, 1.0, true);
+    run_gc_simulation(10, 0.1, 1.5, 1.0, false, false);
```

Comment:
Might be more readable to do a loop here and above
```c
for (full_arg = 0; full_arg < 2; full_arg++) {
    run_gc_simulation(5, 5.0, 2.5, 0.01, false, full_arg);
    // etc
}

Reply:
Good idea. Done.